scrapers/JavBus.yml

```diff
@@ -0,0 +1,104 @@
+name: Javbus
+sceneByFragment:
+  action: scrapeXPath
+  queryURL: https://www.javbus.com/{filename}
+  queryURLReplace:
+    filename:
+      - regex: -JG\d
+        with: ""
+      - regex: (.*[^a-zA-Z0-9])*([a-zA-Z-]+\d+)(.+)
+        with: $2
+  scraper: sceneScraper
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.javbus.com
+      - https://www.seejav.bid
+      - https://www.cdnbus.lol
+      - https://www.dmmbus.lol
+      - https://www.seedmm.cfd
+    scraper: sceneScraper
+sceneByName:
+  action: scrapeXPath
+  queryURL: https://www.javbus.com/search/{}&type=&parent=ce
+  scraper: sceneSearch
+sceneByQueryFragment:
+  action: scrapeXPath
+  queryURL: "{url}"
+  scraper: sceneScraper
+
+performerByURL: 
+  - action: scrapeXPath
+    url: 
+      - https://www.javbus.com
+      - https://www.seejav.bid
+      - https://www.cdnbus.lol
+      - https://www.dmmbus.lol
+      - https://www.seedmm.cfd
+    scraper: performerScraper
+performerByName: 
+  action: scrapeXPath
+  queryURL: https://www.javbus.com/searchstar/{}&type=&parent=ce
+  scraper: performerSearch
+
+xPathScrapers: 
+  performerSearch: 
+    performer: 
+      Name: //span[@class="mleft"]
+      URL: //*[@id="waterfall"]/div/a/@href
+  performerScraper: 
+    performer: 
+      Name: //*[@id="waterfall"]/div[1]/div/div[2]/span
+      Birthdate:
+        selector: //*[@id="waterfall"]/div[1]/div/div[2]/p[contains(text(), '生日')]
+        postProcess:
+          - replace:
+            - regex: ^(.*? ){1}
+              with:
+      Height: 
+        selector: //*[@id="waterfall"]/div[1]/div/div[2]/p[contains(text(), '身高')]
+        postProcess:
+          - replace:
+            - regex: ^(.*? ){1}
+              with:
+      # Measurements: //*[@id="waterfall"]/div[1]/div/div[2]/p[contains(text(), '胸圍')]//*[@id="waterfall"]/div[1]/div/div[2]/p[contains(text(), '腰圍')]//*[@id="waterfall"]/div[1]/div/div[2]/p[contains(text(), '臀圍')]//*[@id="waterfall"]/div[1]/div/div[2]/p[contains(text(), '罩杯')]
```

Comment:
Any reason this is commented out?

Reply:
I haven't found a good way to implement it yet.